modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryNetworkIssuesTest.java

```diff
@@ -383,6 +415,54 @@ private void simulateFailureOfTwoNodes(boolean sequentionally) throws Exception
         }
     }
 
+    /**
+     * This test uses node failure by stopping service threads, which makes the node unresponsive and results in
+     * failing connection to the server.
+     *
+     * @throws Exception If failed.
+     * @see TcpDiscoverySpi#simulateNodeFailure()
+     */
+    @Test
+    public void testCheckNodeFailureSocketConnectionLogMessage() throws Exception {
```

Comment:
Let's make the name more meaningful. Like 'testBackwardConnectionCheckFailedLogMessage'